tuning/trainercontroller/controllermetrics/trainingstate.py

```diff
@@ -49,7 +52,7 @@ def __init__(self, **kwargs):
                 "on_train_begin",
                 "on_evaluate",
             ],
-            **kwargs
+            **kwargs,
```

Comment:
Is this the right convention?

Reply:
seems like it. Here is one [example](https://github.com/foundation-model-stack/fms-hf-tuning/blob/d7280073f8121ff2d0b160302092ddc13010852f/tuning/config/acceleration_configs/acceleration_framework_config.py#L78).